benchmarks/job_runner_benchmark.hpp

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include "cris/core/sched/job_runner.h"
+
+#include <benchmark/benchmark.h>
+
+namespace cris::core {
+
+static void BM_AddJob(benchmark::State& state) {
+    auto job_runner = JobRunner::MakeJobRunner({
+        .thread_num_ = 2,
+    });
+    for ([[maybe_unused]] auto s : state) {
```

Comment:
const?

Reply:
Not sure if it is ok to bind with non-const variable, I will probably just follow what is given by the Google Benchmark user guide.

Reply:
It's always OK to const a var, that's how pass var to f(const) works.
Also that's what we use in cris.